web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -114,31 +114,31 @@ export class LayerDialogComponent implements OnDestroy {
     });
   }
 
+  createNewLayer() {
```

Comment:
Readers [like myself] would assume a `create` methods would implement the factory method design pattern; that they'd be static factories which return a new instance and do not modify state, so this method should update internal state or return a new instance, but not both.